### PR TITLE
unified dockerfiles

### PR DESCRIPTION
**Pull request by Chrisyhjiang**
This PR is designed to make all the base image of node js smaples to be debian:bookworm-slim. 
## Samples Checklist
✅ All good!

### CODE REVIEW

**Comment by Chrisyhjiang** [comment]
Closes #159

**Review by raphaeltm — CHANGES_REQUESTED** [review]
Should use `bookworm-slim` tag but from the official `node` image, not from the debian image.

File: samples/angular-express/backend/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,16 @@
-# Use the official Node.js image
-FROM node:20-alpine
+# Use the slim version of Debian Bookworm as the base image
+FROM debian:bookworm-slim
```
Comment (lionello) [inline samples/angular-express/backend/Dockerfile:2]: [22-bookworm-slim](https://github.com/nodejs/docker-node/blob/c0cfa23e26d75f1f679fd6626a29a27ec5274db3/22/bookworm-slim/Dockerfile)

**Review by lionello — CHANGES_REQUESTED** [review]

File: samples/angular-express/backend/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# Use the official Node.js image
-FROM node:20-alpine
+# Use the slim version of Node.js on Debian Bookworm as the base image
+FROM node:bookworm-slim
```
Comment (lionello) [inline samples/angular-express/backend/Dockerfile:2]: This is better! But we should pick a specific version of Node. Previous sample was using v20, so this should be `20-bookworm-slim`

**Review by lionello — CHANGES_REQUESTED** [review]

File: samples/angular-express/backend/Dockerfile (hunk 2 above)
Comment (Chrisyhjiang) [reply samples/angular-express/backend/Dockerfile]: finished

**Review by lionello — APPROVED** [review]

**Pull request merged**